egs/csj/asr1/conf/tuning/train_pytorch_DC.yaml

```diff
@@ -0,0 +1,53 @@
+# network architecture for Dynamic convolution.
+# encoder related
+elayers: 12
+eunits: 2048
+# decoder related
+dlayers: 6
+dunits: 2048
+# attention related
+adim: 256
+aheads: 4
+
+# hybrid CTC/attention
+mtlalpha: 0.3
+
+# label smoothing
+lsm-weight: 0.1
+
+# minibatch related
+batch-size: 48
+maxlen-in: 512  # if input length  > maxlen-in, batchsize is automatically reduced
+maxlen-out: 150 # if output length > maxlen-out, batchsize is automatically reduced
+
+# optimization related
+sortagrad: 0 # Feed samples from shortest to longest ; -1: enabled for all epochs, 0: disabled, other: enabled for 'other' epochs
+opt: noam
+accum-grad: 13
+grad-clip: 5
+patience: 0
+epochs: 50
+dropout-rate: 0.1
+
+# transformer specific setting
+backend: pytorch
+model-module: "espnet.nets.pytorch_backend.e2e_asr_transformer:E2E"
+transformer-input-layer: conv2d     # encoder architecture type
+transformer-lr: 10.0
+transformer-warmup-steps: 25000
+transformer-attn-dropout-rate: 0.1
+transformer-length-normalized-loss: false
+transformer-init: pytorch
+
+# Ligtweight/Dynamic convolution specific setting
+transformer-encoder-selfattn-layer-type: dynamicconv
+transformer-decoder-selfattn-layer-type: dynamicconv
+wshare: 4
+ldconv-encoder-kernel-length: 101-101-101-101-101-101-101-101-101-101-101-101
```

Comment:
There are multiple format to define a value per layer (e.g: subsample), we should maybe use the same format?